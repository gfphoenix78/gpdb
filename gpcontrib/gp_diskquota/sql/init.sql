\! gpconfig -c shared_preload_libraries -v diskquota > /dev/null
-- end_ignore

\! echo $?

\! gpconfig -c diskquota.monitor_databases -v contrib_regression > /dev/null
-- end_ignore

\! echo $?

-- start_ignore
\! gpstop -rai > /dev/null
-- end_ignore

\! echo $?

\! sleep 10
