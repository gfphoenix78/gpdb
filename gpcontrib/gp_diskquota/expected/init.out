\! gpconfig -c shared_preload_libraries -v diskquota > /dev/null
-- end_ignore
\! echo $?
0
\! gpconfig -c diskquota.monitor_databases -v contrib_regression > /dev/null
-- end_ignore
\! echo $?
0
-- start_ignore
\! gpstop -rai > /dev/null
-- end_ignore
\! echo $?
0
\! sleep 10
